plugins/puppetdb/puppetdb

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python3
+#
+# This code is licensed under GPLv3+
+#
+# Create a graph out of PuppetDB's JVM memory usage.
+#
+# dependencies: python3-requests
+#
+# compatibility:
+#  * PuppetDB 6.x: https://puppet.com/docs/puppetdb/6.0/api/metrics/v1/mbeans.html#jvm-metrics
+#
+# Configuration can be done via environment variables, usually via a munin
+# configuration file for this plugin (e.g.
+# /etc/munin/plugin-conf.d/puppetdb.conf)
+#
+#  * pdburl -- Set the URL to your PuppetDB instance. This url should point to
+#              the mbeans endpoint. By default this has a value of
+#              http://localhost:8080/metrics/v1/mbeans
+#
+#  * timeout -- Time in seconds (int) to wait for a result when querying the REST
+#               API. By default, wait for 2 seconds
+#
+import os
+import sys
+import requests
+
```

Comment:
Just guessing: did you check, whether this plugin follows PEP8? (e.g. run `python3 -m flake8 FILE`)
I think, there should be two empty lines after the imports.